src/lib/dlt_user.c

```diff
@@ -4011,7 +4014,11 @@ DltReturnValue dlt_user_log_send_log(DltContextData *log, const int mtype, int *
     /* send session id */
     if (dlt_user.with_session_id) {
         msg.standardheader->htyp |= DLT_HTYP_WSID;
-        msg.headerextra.seid = (uint32_t) getpid();
+        // 99.99999% of the time, we'll already have a PID set.
```

Comment:
Can you remove this comment? You're right, but it is better to remove this comment.